src/main/sensors/gyro_init.c

```diff
@@ -705,6 +700,10 @@ bool gyroInit(void)
         gyro.accSampleRateHz = 0;
     }
 
+    gyro.scalingAdjustment[X] = 360.0f / (360.0f - gyroConfig()->gyro_scaling_adjustment[X] / 10.0f);
```

Comment:
(all scaling, offset, alignment etc can be factored into 3x4 matrix - like homogeneous coordinates used in graphics)